rdrf/rdrf/forms/components.py

```diff
@@ -266,7 +265,9 @@ def _form(context_form_group):
                 form.heading = _(context_form_group.direct_name)
 
                 form.id = context_form_group.pk
-                form.existing_links = self._get_existing_links(context_form_group)
+                form.existing_links, form.existing_links_index, form.existing_links_len = \
```

Comment:
Minor thing: You might declare existing_links_index and existing_links_len in _Form's constructor.